oswrap/exec.go

```diff
@@ -32,7 +38,42 @@ func (ew execWrap) RunCommandDirect(command []string) error {
 	if err != nil {
 		return err
 	}
-	return cmd.Wait()
+
+	// wait for the command to finish
```

Comment:
another good example is that from [`aws-vault`](https://github.com/99designs/aws-vault/blob/master/cli/exec.go#L179)

Reply:
It would be great to add some tests around this. I know it might be hard though :/